README.md

```diff
@@ -174,13 +174,37 @@ npm run unit
 ```
 
 ## Support
-New Relic hosts and moderates an online forum where you can interact with New Relic employees as well as other customers to get help and share best practices. Like all official New Relic open source projects, there's a related Community topic in the New Relic Explorers Hub. You can find this project's topic/threads here: https://discuss.newrelic.com/c/support-products-agents/node-js-agent/.
+Should you need assistance with New Relic products, you are in good hands with several support channels.
+
+If the issue has been confirmed as a bug or is a feature request, please file a GitHub issue.
+
+**Support Channels**
+
+* [GitHub Documentation](https://newrelic.github.io/node-test-utilities): Test Utilities specific documentation
```

Comment:
Turns out this documentation is identical to the readme. Prob should beef it up sometime if we actually start getting engagement here.